Packs/FiltersAndTransformers/Scripts/DT/DT.yml

```diff
@@ -8,17 +8,18 @@ subtype: python3
 tags:
 - transformer
 - general
-comment: This automation allows the usage of DT scripts within playbooks transformers
+comment: This automation allows the usage of DT scripts within playbooks transformers.
 enabled: true
 args:
 - name: value
   required: true
   description: Value to perform the transformation on.
```

Comment:
The value to perform the transformation on.

Reply:
Fixed, thanks